app/controllers/Authentication.scala

```diff
@@ -37,14 +39,15 @@ import scala.concurrent.Future
 
 
 object AuthForms {
+
   // Sign up
-  case class SignUpData(team:String, email:String, firstName:String, lastName:String, password:String)
+  case class SignUpData(team: String, email: String, firstName: String, lastName: String, password: String)
 
-  def signUpForm(implicit messages:Messages) = Form(mapping(
+  def signUpForm(implicit messages: Messages) = Form(mapping(
     "team" -> text,
     "email" -> email,
     "password1" -> nonEmptyText.verifying(minLength(8)),
-    "password2" -> nonEmptyText,//.verifying(Messages("error.passwordsDontMatch"), password2 => password1 == password2),
+    "password2" -> nonEmptyText, //.verifying(Messages("error.passwordsDontMatch"), password2 => password1 == password2),
```

Comment:
I didn't solve it and I don't know about @rschwanhold

Reply:
Currently it is only solved in the frontend. I can take a look at it in the next days to solve it also on the backend side.

Reply:
Ah ok, I guess its not a high priority then if the frontend checks it.

Reply:
I just recently added a check for the task creation to catch errors, that are not caught by the frontend or are send via a POST request, so I think we should solve that issue. I think it would be good if you look into it @rschwanhold

Reply:
I have added the validation on branch silhouette